### PR TITLE
Parallel EarlGrey Tests

### PR DESCRIPTION
**Pull request by wrumble**
This PR adds passing EarlGrey and XCTest UItests to the WelcomeScreenViewController. They can be run in parallel if you have xcode 9 and iTerm using the ParallelizeTests script found in the root folder.

Im unsure if its necessary to test all the local error messages.

Also unsure how to deal with Firebase error messages as the tests will fail if/when these change despite the code still working correctly.

### CODE REVIEW

File: Splitter/EarlGreyWelcomeScreenTests/EarlGreyWelcomeScreenTests.swift [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    let unregisteredEmail = "unregistered@email.com"
+    let invalidEmail = "invalid@email"
+    
+    override func setUp() {
+        super.setUp()
+        restartApp()
+    }
+    
+    func testHasLogoLabel() {
+        let passwordTextField = grey_accessibilityID(AccesID.passwordTextField)
```
Comment (niamh-power) [inline Splitter/EarlGreyWelcomeScreenTests/EarlGreyWelcomeScreenTests.swift]: Do these not need updating with `AccessID` (the typo fix you've done to the enum :) )

File: Splitter/TestHelpers/TestExtensions.swift [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 import XCTest
 
-enum AccesID {
+enum AccessID {
```
Comment (niamh-power) [inline Splitter/TestHelpers/TestExtensions.swift:11]: All the references to this need updating :)

Comment (wrumble) [reply Splitter/TestHelpers/TestExtensions.swift:11]: Woops. thanks

**Comment by niamh-power** [comment]
You've got some conflicts on stuff you can have in the .gitignore :) example here: https://github.com/github/gitignore/blob/master/Swift.gitignore

...and just noticed that's what you're doing haha :D

**Review by yvettecook — APPROVED** [review]

**Comment by yvettecook** [comment]
Looks good! At some point would love to see a H&T on EarlGrey & UITesting in general.

But this looks good to merge once conflicts fixed

**Review by niamh-power — APPROVED** [review]

**Pull request closed without merging**